TechSupport-Session-Manager/src/main/resources/application.properties

```
@@ -1 +1,3 @@
-
+#server.port=0
```

can now be uncommented
```suggestion
server.port=0
```